test/e2e/common_test.go

```diff
@@ -754,6 +770,18 @@ func SkipIfNotActive(unit string, reason string) {
 	}
 }
 
+func SkipIfNetavark(p *PodmanTestIntegration) {
+	if p.NetworkBackend == Netavark {
+		Skip("This test is not compatible with the netavark network backend")
+	}
+}
+
+func SkipUntilAardvark(p *PodmanTestIntegration) {
```

Comment:
Can you remove this? I think aardvark should cover all dnsname use cases so this should work for all tests

Reply:
@Luap99 lokesh is going to do that in his tests